core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.html

```diff
@@ -13,45 +13,55 @@
     <p-menu [model]="items()" [popup]="true" #menu appendTo="body" data-testId="menu-add"></p-menu>
 }
 
-<div *ngIf="!isContainerEmpty" [ngStyle]="styles.actions" class="actions" data-testId="actions">
-    @if (contentletArea.payload.vtlFiles?.length) {
-        <p-menu [model]="vtlFiles" [popup]="true" #menuVTL appendTo="body" data-testId="menu-vtl" />
-        <p-button
-            (click)="menuVTL.toggle($event)"
-            data-testId="edit-vtl-button"
-            styleClass="p-button-rounded p-button-raised"
-            icon="pi pi-code" />
-    }
-    @if (contentletArea.payload.container) {
+@if (!isContainerEmpty) {
```

Comment:
This file have a *nfIf above
Maybe we can remove it and remove NgIf import from this component
    <p-button
        *ngIf="!isContainerEmpty"
        (click)="setPositionFlag('after'); menu.toggle($event)"
        [ngStyle]="styles.bottomButton"
        data-testId="add-bottom-button"
        icon="pi pi-plus"></p-button>

```suggestion
@if (!isContainerEmpty) { 
        <p-button
        (click)="setPositionFlag('after'); menu.toggle($event)"
        [ngStyle]="styles.bottomButton"
        data-testId="add-bottom-button"
        icon="pi pi-plus"></p-button>
```suggestion
}
```

Reply:
Good catch